aict_tools/io.py

```diff
@@ -550,7 +550,7 @@ def read_cta_dl1(path, aict_config, key=None, columns=None, first=None, last=Non
         df = vstack(tel_dfs)
         if aict_config.disp:
```

Comment:
That's the problem. Why only convert for disp? This is needed for all, right?

Reply:
Oh right, missed it for dxdy.
It was only used in the disp scripts and there are no other units defined in the configs, so this was supposed to copy that behaviour 1 to 1.

Reply:
I wanted to move the unit things directly into the io.
But looks like this doesn't work very well

Reply:
It cant hurt to do it everywhere I suppose